geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -2087,19 +2113,10 @@ void updateSizeOnClearRegion(int sizeBeforeClear) {
     if (isDestroyed || isDestroyingDiskRegion) {
       // If this region is destroyed, mark the stat as destroyed.
       oldMemValue = bytesInMemory.getAndSet(BUCKET_DESTROYED);
-
-    } else if (!isInitialized()) {
-      // This case is rather special. We clear the region if the GII failed.
-      // In the case of bucket regions, we know that there will be no concurrent operations
-      // if GII has failed, because there is not primary. So it's safe to set these
-      // counters to 0.
+    } else {
```

Comment:
This is not quite the same.  Is there a reason why we removed the if check?  We will now always set oldMemValue even if it's been initialized?

why wasn't this just a removal of the else and leave the else if? Just want to understand the logic here a bit...

Reply:
OK, I will keep the "else if" with comments and fix into the "else" only.